### ISSUE TITLE
ED Discovery 9.0.1 causing ED to lag/run slow

### ISSUE THREAD

**Issue by mp1963** [EDDiscovery/EDDiscovery#1580]
I have been heading back to Colonia from the bubble so lots of jumps to be made. A couple of days ago I noticed ED was lagging badly (voice prompts garbled, late or missing, FSD jumps going on for ever etc). I initially thought the ED servers might be playing up but tonight I shut down ED Discovery 9.0.1 and everything in ED went back to normal. The one thing I have noticed is that ED Discovery seems to spend its life waiting for EDSM updates. Perhaps ED Discovery is generating a lot of network/cpu activity which causes ED to slow down ?

**Comment by SW4NS0N** [comment]
I'm getting the same thing, everywhere I go. Makes the game unplayable with the app running. And I REALLY value this app for the fact it gives exploration data guestimated price. I look at the Windows Task Manager and it's mostly harddrive i/o that EDDiscovery is hogging. It's CPU as well, but mostly hd.  A TON OF HD though.

**Comment by imifos** [comment]
I have the same issue and I noted that EDD starts iterating over old screenshoots, always the same, converting it again and again. This behaviour continues even when I deactivated the "convert screenshot" and I pointed the screenshot folder to an empty directory. I conclude that the data it loops over comes from the database and not the directory. In my case, it started to loop each time I landed on a planet causing the game to lag every 5 seconds till I terminated the application.
